src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/Options/ConsoleLoggerFormat.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Diagnostics.Monitoring.TestCommon.Options
+{
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum ConsoleLoggerFormat
+    {
+        [Display(
```

Comment:
These display strings are not generated into the schema.json file. I would remove them so as to not imply that they are having some effect.